packages/core/src/actors/callback.ts

```diff
@@ -171,7 +176,8 @@ export function fromCallback<
         receive: (listener) => {
           callbackState.receivers ??= new Set();
           callbackState.receivers.add(listener);
-        }
+        },
+        emit
```

Comment:
could you add a test that would successfully `emit` from a *rehydrated* callback actor?

Reply:
I will do this in a separate PR

Reply:
1. create a machine with a callback actor
2. persist the root root actor
3. rehydrate a new instance of the root actor using the persisted snapshot
4. now verify that the *rehydrated* actor can correctly use `emit` after step 3

Reply:
53c3c4dd58d29cf003959ffbd5ae279a01cd2ece